_boards_cm/cm4-3d-printer.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: board
+title: "Compute Module 4 Powered 3D Printer Board"
+short_description: CM4 powered CNC board.
+status: prototype
+picture: "/images/board-cm4-3d-printer.png"
+github_issue: "https://github.com/geerlingguy/raspberry-pi-pcie-devices/issues/25#issue-743990222"
+link: https://www.element14.com/community/docs/DOC-96583/l/episode-496-compute-module-4-powered-3d-printer-board
+videos:
+  - https://youtu.be/cFzY_oXxi4A
+---
+The Compute Module 4 Powered 3D Printer Board, created by Clem from Element14 Presents, includes Trinamic Stepper drivers, an ESP32-S2, an ATMega2560-16AU, and many other components. The board allows you to use a CM4 to control the 3D printer's motors, rather than using a computer.
```

Comment:
```suggestion
The Compute Module 4 Powered 3D Printer Board, created by Clem ([@MAYERMAKES](https://twitter.com/MAYERMAKES) from _Element14 Presents_, includes Trinamic Stepper drivers, an ESP32-S2, an ATMega2560-16AU, and many other components. The board allows you to use a CM4 to control the 3D printer's motors, rather than using a computer.
```